### PR TITLE
qlog: update docs and bump to 0.6.0

### PR DESCRIPTION
**Pull request by LPardue**

### CODE REVIEW

**Comment by LPardue** [comment]
builds failing on quiche not being able to pull qlog 0.6.0 from crates.io

**Comment by LPardue** [comment]
fixed now I published it :)

File: qlog/README.md [hunk 1]
```diff
@@ -120,55 +113,59 @@ serde_json::to_string_pretty(&trace).unwrap();
...
+## Streaming Traces JSON Text Sequences (JSON-SEQ)
+To help support streaming serialization of qlogs,
+draft-ietf-quic-qlog-main-schema-01 introduced support for RFC 7464 JSON
+Text Sequences (JSON-SEQ). The qlog crate supports this format and provides
+utilities that aid streaming.
+A [`TraceSeq`] contains metadata such as the [`VantagePoint`] of capture and
+the [`Configuration`]. However, protocol event data is handled as separate
+lines containing a record separator character, a serialized [`Event`], and a
+newline.
+### Creating a TraceSeq
```
Comment (ghedo) [inline qlog/README.md:166]:
```suggestion

### Creating a TraceSeq

```

File: qlog/README.md [hunk 2]
```diff
@@ -120,55 +113,59 @@ serde_json::to_string_pretty(&trace).unwrap();
...
 ```
 
-Streaming Mode
----------------
-
-Create the trace:
-
-```rust
-let mut trace = qlog::Trace::new(
+## Streaming Traces JSON Text Sequences (JSON-SEQ)
```
Comment (ghedo) [inline qlog/README.md:157]:
```suggestion
## Streaming Traces JSON Text Sequences (JSON-SEQ)

```

File: qlog/README.md [hunk 3]
```diff
@@ -120,55 +113,59 @@ serde_json::to_string_pretty(&trace).unwrap();
...
+To help support streaming serialization of qlogs,
+draft-ietf-quic-qlog-main-schema-01 introduced support for RFC 7464 JSON
+Text Sequences (JSON-SEQ). The qlog crate supports this format and provides
+utilities that aid streaming.
```
Comment (ghedo) [inline qlog/README.md:161]:
```suggestion
utilities that aid streaming.

```

File: qlog/README.md [hunk 4]
```diff
@@ -24,22 +24,14 @@ In practice, a single QUIC connection maps to a single Trace file with one
...
-the output, the streaming mode progressively writes serialized JSON output to a
-writer designated by the application.
-
-Buffered Mode
----------------
-
-Create the trace:
+A [`Trace`] is a single JSON object. It contains metadata such as the
+[`VantagePoint`] of capture and the [`Configuration`], and protocol event
+data in the [`Event`] array.
```
Comment (ghedo) [inline qlog/README.md:31]:
```suggestion
data in the [`Event`] array.

```

File: qlog/README.md [hunk 5]
```diff
@@ -24,22 +24,14 @@ In practice, a single QUIC connection maps to a single Trace file with one
...
+JSON Traces allow applications to appends events to them before eventually
+being serialized as a complete JSON object.
+### Creating a Trace
```
Comment (ghedo) [inline qlog/README.md:34]:
```suggestion

### Creating a Trace
```

**Review by ghedo — CHANGES_REQUESTED** [review]
Looks like some empty lines were missed in the README, otherwise lgtm.

**Review by ghedo — APPROVED** [review]

**Pull request merged**